tutorials/spectrum_pipe.ipynb

```diff
@@ -96,7 +96,7 @@
     "    reference=1 * u.TeV,\n",
```

Comment:
There is `SkyCoord.from_name("crab")` that breaks Travis CI. Maybe this should be removed?

Reply:
Done